Launcher/MainWindow.xaml.cs

```diff
@@ -277,7 +277,7 @@ public MainWindow()
                 }
                 catch (WebException ex) when (ex.InnerException is IOException)
                 {
-                    MessageBox.Show("Updating H2Codez failed because the launcher can't write the update data.\nPlease close all currently open modding tools and try again.", "Error!");
+                    MessageBox.Show("Updating H2Codez failed because the launcher can't write to the update data.\nPlease close all currently open modding tools and try again.", "Error!");
```

Comment:
The issue is writing the update data to disk, not writing to update data.
```suggestion
                    MessageBox.Show("Updating H2Codez failed because the launcher can't save the update data to disk.\nPlease close all currently open modding tools and try again.", "Error!");
```